src/components/screens/EventRecord.js

```diff
@@ -109,6 +114,15 @@ export const EventRecord = ({ data, navigation }) => {
         );
     });
 
+  useMatomoTrackScreenView(
```

Comment:
i like structures!
currently most of the components start with `useState` and `useContext` stuff followed by destructing of the props.
`useEffect` should come also before the rest of the logics. but correct, this is not followed that strict because of different situations in certain components. we can plan this to be refactored all over the codebase. could this be a ticket for you, which could help to get to know the app better?

having everything hook related in one big block, separated in the way you meant, it sounds good to me to destruct props at first and rest of the logic at the end before render return.